src/Tribe/Main.php

```diff
@@ -3054,6 +3057,10 @@ public function publishAssociatedTypes( $post_id, $post ) {
 			remove_action( 'save_post_' . self::VENUE_POST_TYPE, array( $this, 'save_venue_data' ), 16, 2 );
 			remove_action( 'save_post_' . self::ORGANIZER_POST_TYPE, array( $this, 'save_organizer_data' ), 16, 2 );
 
+			// Remove any "preview" venues and organizers (duplicates) attached to this event.
+			$this->remove_preview_venues( $post_id, true );
```

Comment:
Can we attach these to an action, so it's not running on `plugins_loaded` where we don't have a WP User set yet? Maybe `admin_init` or `init`